frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -91,7 +91,7 @@ export default function SideNavigation({
   }, [])
 
   async function fetchReviewCount() {
-    setReviewCount((await getReviewCount()).headers.get('x-count') as unknown as number)
+    setReviewCount((await getReviewCount()).headers.get('x-count') as string)
```

Comment:
I think it would be cleaner to parse it as a number here.  E.g:

```
setReviewCount(parseInt((await getReviewCount()).headers.get('x-count')))
```

Then reviewCount can remain a number, and you can compare it as a number

Reply:
Is it be preferable to raise an error on a null response or set the value to 0?

Reply:
IMO we should probably send some sort of feedback if it can't find it and then set it to 0 so that it hides the bubble. 

We use the `useNotification` hook as defined [here](https://github.com/gchq/Bailo/blob/main/frontend/src/common/Snackbar.tsx) to show snackbar type feedback to user. There's an example of implementation (here)[https://github.com/gchq/Bailo/blob/main/frontend/src/Form/RenderButtons.tsx#L42-L46].